packages/gatsby-theme-newrelic/src/components/Banner.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/core';
+import Button from './Button';
+import Icon from './Icon';
+
+const Banner = ({ children, color }) => {
+  const [visible, updateVisible] = useState(true);
+
+  if (!visible) return null;
+
+  return (
+    <div
+      css={css`
+        height: 90px;
```

Comment:
Something to consider here is that this `90px` locks us into a specific size for content regardless of how much text we have inside the banner. We may also have text overflow issues for smaller screens if there is a decent amount of text inside the banner. Should we instead let the banner use its natural height? Or maybe consider making this a `min-height`?

Reply:
@jpvajda @djsauble might have some opinions about this, we were just following the acceptance criteria from the issue! linked [here](https://github.com/newrelic/developer-website/issues/615)

Reply:
The banner should be responsive to text input ideally. 90px is just a standard for banners, but if someone exceed thats limit the banner ideally should accommodate the test.

Reply:
How do images work here? If I want a full-bleed background image, is that possible? If so we might need users to be able to provide height as a prop.

Reply:
I think switching this to `min-height` should solve all of these scenarios (unless we want something shorter for single-line text). That way if there are taller images, it would grow with it.